BambeeHR/pollen
# feat (ARRR-373) Nav Icons for Employee Voices

### Description
https://bambee.atlassian.net/jira/software/projects/ARRR/boards/63?selectedIssue=ARRR-373

This PR adds the megaphone Icons: `megaphone.svg, megaphone-solid.svg` to the pollen `BaseIcon` and `HighlightedIcon`.

### Screenshot
![Screenshot from 2021-09-28 13-45-24](https://user-images.githubusercontent.com/10406785/135163006-9a155f90-f6e2-4827-8628-0927e0a1dc2c.png)
![Screenshot from 2021-09-28 13-45-12](https://user-images.githubusercontent.com/10406785/135163008-517a0bcb-ab6b-47f9-97de-5cabe9e610b4.png)


### Manual Testing
1. `yarn dev`
2. go to BaseIcon and see megaphone and megaphoneSolid
3. go to HighlightedIcon and see megaphone and megaphoneSolid.
